update-common/src/artifacts/artifacts_with_plan.rs

```diff
@@ -332,6 +345,16 @@ pub enum ControlPlaneZonesMode {
     Split,
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum VerificationMode<'a> {
+    /// Verify the uploaded repository is accepted by one of these trusted root
+    /// roles.
+    TrustStore(&'a [Vec<u8>]),
+    /// Blindly trust the root role present in the repository (but still perform
+    /// signature checks using that root role).
+    BlindlyTrustAnything,
```

Comment:
I regret that this exists but I gather we need it for MUPdate and also use it in `reconfigurator-cli`.

Reply:
Yeah. I think eventually we'll add some sort of trust to Wicket but we should design what we want there.